pkg/model/iam/iam_builder.go

```diff
@@ -162,25 +145,12 @@ func (b *IAMPolicyBuilder) BuildAWSIAMPolicy() (*IAMPolicy, error) {
 		}
 
 		if kmsKeyIDs.Len() > 0 {
-			p.Statement = append(p.Statement, &IAMStatement{
-				Effect: IAMStatementEffectAllow,
-				Action: stringorslice.Of(
-					"kms:Encrypt",
-					"kms:Decrypt",
-					"kms:ReEncrypt*",
-					"kms:GenerateDataKey*",
-					"kms:DescribeKey",
-					"kms:CreateGrant",
-					"kms:ListGrants",
-					"kms:RevokeGrant",
-				),
-				Resource: stringorslice.Slice(kmsKeyIDs.List()),
-			})
+			addKMSIAMPolicies(p, stringorslice.Slice(kmsKeyIDs.List()))
 		}
-	}
 
-	if b.HostedZoneID != "" {
-		addRoute53Permissions(p, b.HostedZoneID)
+		if b.HostedZoneID != "" {
```

Comment:
This is a behaviour change, right?  We are no longer adding route53 permissions to the nodes?  That should be called out - we've broken people before with this!